doc/api/addons.md

```diff
@@ -1,6 +1,6 @@
 # C/C++ Addons
```

Comment:
The N-API actually supports writing Addons ons in C so C/C++ is correct for them.  The references to  addons.html of course need to be updated everywhere.